aspnetcore/tutorials/first-web-api.md

```diff
@@ -98,19 +96,17 @@ A NuGet package must be added to support the database used in this tutorial.
 
 # [Visual Studio for Mac](#tab/visual-studio-mac)
 
-For Visual Studio for Mac, switch to the .NET 7 version of this tutorial.
-
-<!-- * In Visual Studio for Mac 2022, select **File** > **New Project...**.
+ * In Visual Studio for Mac 2022, select **File** > **New Project...**.
 
 * In the **Choose a template for your new project** dialog:
   * Select **Web and Console** > **App** > **API**.
   * Select **Continue**.
 
 * In the **Configure your new API** dialog, make the following selections:
-  * Confirm the **Target framework** is **.NET 7.0** (or later).
-  * Confirm the checkbox for **Use controllers (uncheck to use minimal APIs)** is checked.
+  * Confirm the **Target framework** is **.NET 8.0** (or later).
```

Comment:
I'm not sure "or later" makes sense here. We don't say that in the prerequisites section, and we don't know what changes may come in .NET 9.

Reply:
Corrected this. Thanks Tom!